PersonStageThree/src/com/beyond/person/entrypoint/Main.java

```diff
@@ -0,0 +1,59 @@
+package com.beyond.person.entrypoint;
+
+import com.beyond.person.core.exception.InvalidFieldException;
+import com.beyond.person.helper.InvalidPersonException;
+import com.beyond.person.helper.SetterClass;
+import com.beyond.person.helper.UserInput;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        int count = 0;
+        String input = "";
+        Scanner enter = new Scanner(System.in);
+        System.out.println("Hello! Let's create a person");
+        while (count < 3) {
+            System.out.println("What kind of person do you want to create? 1) dancer  2) singer  3) programmer ");
+            System.out.println("Or choose 4) exit to terminate the program");
+            input = enter.nextLine();
+            try {
+                if (!input.equals("4")) {
```

Comment:
@maria-salmasian what mean this numbers ? why you don't use enum ?

Reply:
changed it to enums